### PR TITLE
Use released docker-java version

### PR DESCRIPTION
**Pull request by guilhem**
/!\ error in build process

### CODE REVIEW

**Comment by guilhem** [comment]
https://gist.github.com/guilhem/59fe9decf673ae1e8a4f

**Comment by thomassuckow** [comment]
Should probably also think about adding the docker-java snapshot repository to the pom.

File: pom.xml [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.nirima.docker-java</groupId>
+            <groupId>com.github.docker-java</groupId>
```
Comment (KostyaSha) [inline pom.xml]: This fix seems right

File: pom.xml [hunk 2]
```diff
@@ -24,9 +24,9 @@
...
             <artifactId>docker-java</artifactId>
-            <version>0.10.5-SNAPSHOT</version>
+            <version>0.10.5</version>
```
Comment (KostyaSha) [inline pom.xml]: This wrong, SNAPSHOT version for not released plugin version is ok, because it needs fixes for functionality.

Comment (guilhem) [inline pom.xml]: `0.10.5` version is released... if we want another one, we may use `0.10.6-SNAPSHOT`

Comment (KostyaSha) [inline pom.xml]: Right, then ok

**Comment by imasen** [comment]
Could someone upload the jar for the version of  @magnayn ? It's hard to develop with red lines in IDE.

**Comment by thomassuckow** [comment]
His is in OSS
 <repository>
            <id>oss</id>
            <url>http://oss.sonatype.org/content/groups/public/</url>
            <releases>
               <enabled>false</enabled>
            </releases>
            <snapshots>
               <enabled>true</enabled>
            </snapshots>
        </repository>

**Comment by magnayn** [comment]
My changes are also in github - I'm slowly sending PRs to docker-java to
try to get back to mainline.

**Comment by KostyaSha** [comment]
@magnayn please fix master or branch latest released version so people may continue fixing their problems. You can release your any future not tested features as beta release from master then.

**Comment by CedricGatay** [comment]
Adding the repository field worked for me to be able to build properly locally.

**Comment by magnayn** [comment]
N/A.

We are nearly back to mainline. When that happens we can flip back.

**Pull request closed without merging**